### DISCUSSION TITLE
Enable download again?

### DISCUSSION THREAD

**Discussion by pierrelasse** [YimMenu/YimMenu#2141] [General]
So the download is down because players got banned but now that there is a "fix", can it please be enabled  again?
Because im banned and i have nothing to loose anymore and i would like to use the new features.
Now ppl are using the old version that bans instead  of the new one that only maybe bans.
https://github.com/YimMenu/YimMenu/releases

(writing the second time bec im stupid : D)

**Comment by iDominiqe** [comment]
![image](https://github.com/YimMenu/YimMenu/assets/115355282/58fdf4c6-e673-4fd2-8eec-e181edf69d48)

**Reply by pierrelasse** [reply]
you can post it here https://github.com/YimMenu/YimMenu/issues/2124

**Comment by ghost** [comment]
> So the download is down because players got banned but now that there is a "fix", can it please be enabled again? Because im banned and i have nothing to loose anymore and i would like to use the new features. Now ppl are using the old version that bans instead of the new one that only maybe bans. https://github.com/YimMenu/YimMenu/releases
> 
> (writing the second time bec im stupid : D)

![yimmenu_how_to_dl_without_releases](https://github.com/YimMenu/YimMenu/assets/122435102/58bef73f-260e-4416-b498-f9b1c2d70fe0)

**Reply by pierrelasse** [reply]
binary.zip sounds like it contains the compiled code or src but not the final dll.

**Reply by Yimura** [reply]
Doesn’t hurt to download and check what’s inside.

**Reply by pierrelasse** [reply]
Thx for the info anyways @freemantsinq.
Sorry for not checking everything in every pull and commit and download everything i see even when assuming it only contains the code related to the pull request and not the built dll @Yimura.
Also doesn't hurt to link releases in the readme and maybe add more info about the current situation in the readme.
